webapi/Services/IOcrEngine.cs

```diff
@@ -10,7 +10,6 @@ namespace SemanticKernel.Service.Services;
 /// </summary>
 public interface IOcrEngine
 {
-
```

Comment:
Intentional?  (Perhaps this file can be reverted)

Reply:
I didn't edit this file, so this was a dotnet format autofix -- will leave as is